src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -315,9 +315,10 @@ var ReactComponentTreeHook = {
     }
 
     var currentOwner = ReactCurrentOwner.current;
-    var id = currentOwner && currentOwner._debugID;
-
-    info += ReactComponentTreeHook.getStackAddendumByID(id);
+    if (currentOwner && typeof currentOwner._debugID === 'number') {
+      var id = currentOwner && currentOwner._debugID;
+      info += ReactComponentTreeHook.getStackAddendumByID(id);
+    }
```

Comment:
Yeah tbh I barely looked at what this file does; just trying to get it merged for now :D